StatusBar.c

```
@@ -0,0 +1,190 @@
+   if (len > 8 && String_eq(buffer + len - 8, " thermal"))
+      buffer[len - 8] = '\0';
+
+   for (char* token = buffer; *token;) {
+      while (*token == ' ')
+         token++;
+      if (!*token)
+         break;
+
+      char* end = token;
+      while (*end && *end != ' ')
+         end++;
```

Cppcheck (2.21.0)

[style] The function 'HeaderLayout_getColumns' is never used.

(unusedFunction)